src/components/15Years/15YearsIndividualsData.js

```diff
@@ -321,26 +321,26 @@ export default [
 		link2: '/monthlygood?category=refugees',
 	},
 	{
-		name: 'Lihn',
+		name: 'Linh',
```

Comment:
Already noted in slack, but I'll mention it here too, **Lihn** is the correct spelling, not Linh.